build/docker/edgeimg-make.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+REPOSITORY=${REPOSITORY:-"kubeedge/edgecore"}
```

Comment:
Let's named the repo to `kubeedge/installation-package` :)  And place the keadm in `/usr/local/bin/`.